br-ext/package/optee_client_ext/Config.in

```diff
@@ -10,7 +10,7 @@ config BR2_PACKAGE_OPTEE_CLIENT_EXT_SITE
 	  The path to the source directory of OP-TEE client
 
 config BR2_PACKAGE_OPTEE_CLIENT_EXT_RPMB_EMU
-	bool "Enable real RPMB device"
+	bool "enable RPMB emulation"
```

Comment:
Upper case "Enable" ;)

Reply:
I asked that myself before updating this P-R. Looking into buildroot source tree, the descriptions tend to use lower case 1st char in the config switch short description.
You prefer an upper case char here?

Reply:
Not so clear cut... (considering only multi-word description to exclude simple package or component names which are always written lowercase)
```
buildroot ((2021.02))$ find . -name Config.in | xargs egrep '(bool|int|string).*\"[a-z].* .*' | wc -l
739
buildroot ((2021.02))$ find . -name Config.in | xargs egrep '(bool|int|string).*\"[A-Z].* .*' | wc -l
924
```
According to the same commands, the Linux kernel overwhelmingly uses capitalization (94.8 % of the time to be precise ;-)).

Reply:
fair